src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,22 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+def _check_call_error(call_error, metadata):
+  if call_error == cygrpc.CallError.ok:
+    return
+  elif call_error == cygrpc.CallError.invalid_metadata:
+    raise ValueError("metadata was invalid: %s" % metadata)
+  else:
+    raise ValueError('Internal gRPC CallError %d. Please report to '
```

Comment:
Avoid implicit string concatenation whenever you can.

If our issue tracker URL is going to appear twice in program source, define a constant for it?